components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -408,9 +424,14 @@ - (void)viewDidLoad {
     [self addChildViewController:self.contentViewController];
     [self.scrollView addSubview:self.contentViewController.view];
     [self.contentViewController didMoveToParentViewController:self];
+    [self.scrollView insertSubview:self.shadowedView atIndex:0];
   }
 
   self.scrollView.accessibilityIdentifier = kMDCBottomDrawerScrollViewAccessibilityIdentifier;
+
+  self.shadowedView.layer.shadowColor = self.drawerShadowColor.CGColor;
+  self.shadowedView.backgroundColor = UIColor.clearColor;
+  ((MDCShadowLayer *)self.shadowedView.layer).elevation = self.elevation;
```

Comment:
Tip: Consider exposing a property called `shadowLayer` in `MDCBottomDrawerShadowedView` so you don't have to cast here. It keeps the code safer for changes in the future.

Reply:
Done.